tests/pkg_winc1500_netdev/README.md

```diff
@@ -0,0 +1,32 @@
+# WINC1500 Wifi Module Driver
+
+## How to use the test program
```

Comment:
Maybe also add a subsection under this with minimal examples of shell commands used (scan, connect to an AP, etc).